add-ons/grafana/grafana.addon

```diff
@@ -0,0 +1,16 @@
+# Name: grafana
+# Description: This template adds grafana for displaying Openshift Metrics in its own namespace. If requires metrics and management-infra addon, to connect to hawkular-metrics on openshift.
+# Url: https://raw.githubusercontent.com/hawkular/hawkular-grafana-datasource/master/openshift/openshift-template-persistent.yml
+# Required-Vars: namespace 
+
+# TODO: Once conditional commands are there, create the namespace if it does not exist
+oc adm new-project #{namespace}
+oc new-app -f openshift-template-persistent.yml -n #{namespace}
+
+echo You have installed #{addon-name}
+echo To access #{addon-name} go to https://hawkular-grafana-#{namespace}.#{routing-suffix}
```

Comment:
This one is giving me 503  Service unavailable
```
$ curl -I https://hawkular-grafana-grafana.192.168.42.82.nip.io --insecure
HTTP/1.0 503 Service Unavailable
Pragma: no-cache
Cache-Control: private, max-age=0, no-cache, no-store
Connection: close
Content-Type: text/html
```

For *http* I am able to access: 
```
$ curl -I http://hawkular-grafana-grafana.192.168.42.82.nip.io
HTTP/1.1 302 Found
Location: /login
Set-Cookie: grafana_sess=c1bd54b73014dda2; Path=/; HttpOnly
Set-Cookie: redirect_to=%252F; Path=/
Date: Wed, 30 Aug 2017 13:20:45 GMT
Content-Type: text/plain; charset=utf-8
Set-Cookie: fc03a0e6557d5577400af0d9f8556ff9=91a33622f1c878e3ac9f698fcb2ea388; path=/; HttpOnly
```
Also, via `minishift openshift service hawkular-grafana -n grafana --in-browser` which opens in *http* scheme.

Reply:
Fixed http/https issue in new commit.